src/nagbot.ts

```diff
@@ -221,16 +228,38 @@ export class NagBot {
     }
 }
 
-function generateSecretKey(length: number = 16): string {
-    let buf = randomBytes(length);
-    return buf.toString('hex');
-}
-
-function guid() { return generateSecretKey(32); }
-
 const helpMessage = `I am NagBot.
 
 You can ask me to do any of the following:
 * Clear channels
 * Create a reminder; e.g. remind me to walk the dog tomorrow noon
 * List reminders: what are my reminders?`;
+
+function ProcessDateTimeEntity(dateEntity: { type: string, timex: string[] }[]) {
+    // "https://github.com/Microsoft/Recognizers-Text/blob/master/JavaScript/packages/recognizers-date-time/src/dateTime/constants.ts"
```

Comment:
What are these two lines? There's another further down.

Reply:
These are links to where the Timex code gets processed in bot samples.  Important to keep these around since there is no doc. :-(